### ISSUE TITLE
Can I use latest version with requirejs?

### ISSUE THREAD

**Issue by marcgise** [stomp-js/stompjs#360]
Dear, I'm using an old version of StompJS with RequireJS, and I would like to use this latest version, but I can't achieve that. Is it possible to do that?
Thanks,

**Comment by kum-deepak** [comment]
I do not use RequireJS myself. The packaging is UMD so it should be usable with RequireJS. Please provide me with the following:
-  Which was the version you were using earlier?
- Share with me a sample page so that I can see the error it gives.

**Comment by marcgise** [comment]
Hi, I don't know the version of Stomp I used before, because it doesn't appear in the file... The only information that appears, is that it's generated with CoffeeScript 1.7.1. I'm using Stomp and SockJs in combination with Spring to use WebSockets, but I want to upgrade the version of Stomp to the latest.
If I create a single page importing the scripts with html from CDN, it works fine. But if I tried to use RequireJs to load it (our application uses RequireJs to build), it doesn't work.
`
define(["stomp"], function(stomp) {
.............
});
`
